runtime/doc/lua.txt

```diff
@@ -4308,4 +4308,46 @@ vim.text.hexencode({str})                               *vim.text.hexencode()*
     Return: ~
         (`string`) Hex encoded string
 
+
+==============================================================================
+Lua module: tohtml                                                    *tohtml*
+
+tohtml({winid}, {outfile}, {opt})                            *tohtml.tohtml()*
+    Converts a window to HTML.
+
+    Converts a window to HTML, including diffs and folds. Opens a slit window
+    containing the converted HTML.                           *convert-to-HTML*
+
+    Can also be converted with `:TOhtml` .                           *:TOhtml*
+
+    Parameters: ~
+      • {winid}    (`integer?`) Defaults to current window.
+      • {outfile}  (`string?`) Defaults to a temp file.
+      • {opt}      (`table?`) Optional parameters.
+                   • title (string): Title of the html (defaults to buffer
+                     name)
+                   • number_lines (boolean): Show line numbers (defaults to
+                     `false`)
+                   • font (string|string[]): Fonts to use (defaults to
+                     `guifont`)
+                   • width (integer): Width of the html (defaults to
+                     'textwidth' or window width)
+
+tohtml_lines({winid}, {opt})                           *tohtml.tohtml_lines()*
```

Comment:
I don't think we need two functions here, one should be enough. Adding a light wrapper that just writes to a file isn't worth extra surface area IMO.

From what I understand, the goal is provide a `:Tohtml` ex command. If so, the ex command should take an output file name as an argument, and internally use `vim.tohtml()` which just outputs a string.